### ISSUE TITLE
Update the API for retrieving tee time rates for "as low as"

### ISSUE THREAD

**Issue by robinneb** [robinneb/golfzing#16]
Task: Update the endpoint for retrieving tee times to include "as low as"

Specifications: Ashley, please refer to the last email conversation with Drew about the API accommodating a logic of "as low as" for retrieving dollar amounts for tee times as well as Zing Points "as low as". We will need to add this to the experience but need an API built for it. You can think through the details and we can go over it together]

**Comment by drubage** [comment]
Can you confirm what data you will pass to us when making that call? Like a start/end time? Or a "day part"? A date or are we looking at all dates? Will you be sending us the course ID? Thanks!

**Comment by amatsui** [comment]
@drubage, Yes. we want to also have it be combined with endpoints such as players, time of day, etc. 
for example, if I want 18 holes at 6pm for 3 people, the "as low as" amount should reflect that criteria if I filtered for it in the app. 

@digitalbrandgroup, Will we send Fruition the Course ID? How will this work?

**Comment by drubage** [comment]
@digitalbrandgroup @amatsui Did you all see the notes here?

https://github.com/robinneb/golfzing/issues/28

We were hoping you could help test the feasibility of loading the entire giant tee sheet into the app's database (or at least larger chunks of it at a time). Were you able to experiment with that at all? If that works then all logic for showing lowest/highest/average could be done in the app based off the real data (we can help you formulate the queries as necessary) and wouldn't require additional API methods or features on the Golfzing side. If the data cannot be loaded because it is too big then can you help determine the maximum size the tee time database in the app could get to see if there's ways we could minimize the data coming across from the feed and still get all the tee time data into the app?

Let us know if you can, thank you!!!
